docs/user/filesystem/commandline.rst

```diff
@@ -120,7 +120,7 @@ Argument                 Description
 ``--leaf-storage``       Use leaf storage
 ``--target-file-size``   Target size for data files (e.g. 500MB or 1GB)
 ``--temp-path``          Path to a temp directory used for working files
-``--storage-opt``        Additional storage options, as ``key=value``
+``--storage-opt``        Additional storage options to set as SimpleFeatureType user data, in the form ``key=value``
```

Comment:
Is there a quick example of a storage-opt which has a comma in it?

Reply:
some of the storage opts are json, so they have commas. did you mean for including in the docs?

Reply:
Eh, knowing that it is json is helpful.  If adding to the docs is handy, feel free to do it later.  Merging!